.github/workflows/test.yml

```diff
@@ -310,6 +315,7 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
           workflow: test.yml
+          workflow_conclusion: ''
```

Comment:
Is this correct? Shouldn't it be `success` or some other value specified in https://github.com/dawidd6/action-download-artifact#usage?

Reply:
If it ever reaches this point https://github.com/ZcashFoundation/zebra/pull/3556/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R275 it was because the regeneration was successful.

Sometimes there's an error or a timeout in the following steps, and the whole workflow throw doesn't throw a `success`, but the generated disk is still valid, and we want that disk to be used in the following run